hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/DeleteMetadataActor.scala

```diff
@@ -11,26 +12,32 @@ import cromwell.services.metadata.hybridcarbonite.DeleteMetadataActor.DeleteMeta
 import cromwell.services.metadata.impl.MetadataDatabaseAccess
 
 import scala.util.{Failure, Success}
-import scala.concurrent.duration._
 
-class DeleteMetadataActor(metadataDeletionConfig: MetadataDeletionConfig) extends Actor
+class DeleteMetadataActor(metadataDeletionConfig: MetadataDeletionConfig) extends Actor with StrictLogging
   with ActorLogging
   with MetadataDatabaseAccess
   with MetadataServicesStore {
 
   implicit val ec = context.dispatcher
 
-  metadataDeletionConfig.intervalOpt.map { interval =>
-    context.system.scheduler.schedule(5.minutes, interval, self, DeleteMetadataAction)
+  var activeConfig: ActiveMetadataDeletionConfig = _
```

Comment:
ew

Reply:
Strange to have an actor be created and then immediately shut itself down. How about having the would-be parent of this actor not create this actor if the deletion config is inactive?